### ISSUE TITLE
Unable to resolve module `AccessibilityInfo`

### ISSUE THREAD

**Issue by Amurmurmur** [mfrachet/rn-placeholder#9]
Once this plugin is installed the following exception gets thrown:
```
Unable to resolve module `AccessibilityInfo` from `/Users/amurmurmur/examplepp/node_modules/react-native/Libraries/react-native/react-native-implementation.js`: Module does not exist in the module map

This might be related to https://github.com/facebook/react-native/issues/4968
To resolve try the following:
  1. Clear watchman watches: `watchman watch-del-all`.
  2. Delete the `node_modules` folder: `rm -rf node_modules && npm install`.
  3. Reset packager cache: `rm -fr $TMPDIR/react-*` or `npm start -- --reset-cache`.

RCTFatal
__28-[RCTCxxBridge handleError:]_block_invoke
_dispatch_call_block_and_release
_dispatch_client_callout
_dispatch_main_queue_callback_4CF
__CFRUNLOOP_IS_SERVICING_THE_MAIN_DISPATCH_QUEUE__
__CFRunLoopRun
CFRunLoopRunSpecific
GSEventRunModal
UIApplicationMain
main
start
0x0
```

RN v0.46.1
rn-placeholder v0.0.7 (but says 0.0.6 in the package.conf)

Hopefully somebody helps 👍

**Comment by mfrachet** [comment]
Thanks for having opened this issue.

It seems to be a problem with the react native version you use and the one I use inside of this package. 

Can you try with RN v0.44 and tell me if it's always happening ?


For now, I m going to investigate on that versions diff and wish to propose a solution to that problem.

**Comment by mfrachet** [comment]
I've taken a look at some react native issues, and I've found this one pointing out a problem with react-native or npm [react-native#14209](https://github.com/facebook/react-native/issues/14209)

I think the next major release of react-native would provide à way to manage this kind of errors. Npm seems also to be working on the problem.

I m going to follow the thread and provide some improvement to the project. But by now, if you want to use this library, you would have to use a react-native version before 0.44.2

Thanks for having opened that issue, and feel free to help again a next time 👍

**Comment by mfrachet** [comment]
Problem solved in 0.0.7, related PR : https://github.com/mfrachet/rn-placeholder/pull/11
